server/utils/tasks/index.ts

```diff
@@ -70,6 +71,15 @@ const taskSummary = (task: Task, application: Application): Array<SummaryListIte
       value: {
         text: getApplicationType(application),
       },
+      actions: {
+        items: [
+          {
+            href: `${paths.applications.show({ id: application.id })}?tab=timeline`,
+            text: 'View timeline',
+            visuallyHiddenText: 'timeline',
```

Comment:
I don't think this line is needed - a screenreader user would head "View timeline timeline" which doesn't seem right